tooling/src/main/java/org/opencds/cqf/tooling/common/ThreadUtils.java

```diff
@@ -16,6 +16,8 @@ public class ThreadUtils {
 
     private static List<ExecutorService> runningExecutors = new ArrayList<>();
 
+    private static final int threadPoolCount = Runtime.getRuntime().availableProcessors() * 2;
```

Comment:
Using a magic number heuristic here. It makes sense in context, but I would prefer a safer approach, like the following:

`private static final int threadPoolCount = Integer.getInteger("threadutils.pool.count", Runtime.getRuntime().availableProcessors());`

This allows configuration to override e.g. `-Dthreadutils.pool.count=8`

Reply:
> Using a magic number heuristic here. It makes sense in context, but I would prefer a safer approach, like the following:
> 
> `private static final int threadPoolCount = Integer.getInteger("threadutils.pool.count", Runtime.getRuntime().availableProcessors());`
> 
> This allows configuration to override e.g. `-Dthreadutils.pool.count=8`

I do like this better. Thank you!